src/components/Gameplay.js

```diff
@@ -9,25 +9,25 @@ const Gameplay = ({ getRulesLinks }) => {
         <section className='main__section'>
           <h3 className='main__subheading'>Spellcasting</h3>
           <div className='main__links'>
-            {getRulesLinks('spellcasting')}
+            {getLinks('gameplay-basics', 'spellcasting')}
           </div>
         </section>
         <section className='main__section'>
           <h3 className='main__subheading'>Adventuring</h3>
           <div className='main__links'>
-            {getRulesLinks('adventuring')}
+            {getLinks('gameplay-basics', 'adventuring')}
           </div>
         </section>
         <section className='main__section'>
           <h3 className='main__subheading'>Combat</h3>
           <div className='main__links'>
-            {getRulesLinks('combat')}
+            {getLinks('gameplay-basics', 'combat')}
           </div>
         </section>
         <section className='main__section'>
           <h3 className='main__subheading'>Ability Scores</h3>
           <div className='main__links'>
-            {getRulesLinks('using ability scores')}
+            {getLinks('gameplay-basics', 'using ability scores')}
           </div>
         </section>
```

Comment:
the structure inside the main tags also get repeated quite often. I think there is another opportunity here for a larger component that maybe accepts a `mainHeading`, an array of sections, and iterates over them to generate `<LinkSections />` like mentioned before. This might allow you to eliminate multiple components

Reply:
Added to existing issue for directories